components/automate-ui/package-lock.json

```diff
@@ -612,7 +612,8 @@
             "ansi-regex": {
               "version": "2.1.1",
               "bundled": true,
-              "dev": true
+              "dev": true,
+              "optional": true
```

Comment:
All package-lock.json changes are adding the `optional` property to a large number of packages.